data_steward/validation/metrics/required_labs.py

```diff
@@ -15,6 +21,10 @@
 MEASUREMENT_CONCEPT_SETS_TABLE = '_measurement_concept_sets'
 MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE = '_measurement_concept_sets_descendants'
 
+ROW_COUNT_QUERY = JINJA_ENV.from_string("""SELECT table_id, row_count
+FROM `{{project_id}}.{{dataset_id}}.__TABLES__`
+WHERE table_id IN ('concept', 'concept_ancestor')""")
```

Comment:
You don't have to add a qualifier for the concept/concept_ancestor here. This will help in determining whether the `MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE` table exists